marketplace/manifest.yaml

```diff
@@ -1,12 +1,12 @@
-applicationName: com.spinkube.spinkube
-publisher: "Spinkube"
+applicationName: spinkube
+publisher: "Fermyon"
 description: "SpinKube on Azure Marketplace"
-version: 1.0.0 #Must be in the format of #.#.#
+version: 1.0.4 #Must be in the format of #.#.#
```

Comment:
FYI: This has been bumped while iterating on a preview offering.  The Azure Marketplace prohibits version re-use, necessitating a version rev on any change.

Reply:
Updated to version `0.1.0` to signal to users managing the extension that it is pre-release (as spin operator and thus spinkube is)

Reply:
_...annnddd_ [we're back to 1.0.5](https://github.com/spinkube/azure/pull/4/commits/7a98e7796b78174445005ab43b43d59e55eac4e7) as the Marketplace backend doesn't support reverting to an earlier version.

As a follow up I'll look into the feasibility of offering users a chance to select the underlying SpinKube version when installing the offering, to decouple from this bundle version.